src/IMetrom.sol

```diff
@@ -37,36 +37,76 @@ struct RewardsCampaign {
 /// This can optionally contain a SHA256 hash of some JSON content stored on IPFS such
 /// that a CID can be constructed from it. `specification` can point to an IPFS JSON
 /// file with additional information/parameters on the campaign.
-struct PointsCampaign {
+struct PointsCampaignV1 {
     address owner;
     address pendingOwner;
     address pool;
     uint32 from;
     uint32 to;
-    bytes32 specification;
+    bytes32 specificationHash;
+    uint256 points;
+}
+
+/// @notice Represents a rewards based campaign in the contract's state, with its owner,
+/// running period, type, data, specification, root and data links, as well as rewards
```

Comment:
Minor change to the comment, for `specificationHash` and `dataHash`, also in the above structs.